nvflare/fuel/f3/cellnet/net_manager.py

```diff
@@ -29,8 +29,9 @@ def _to_int(s: str):
 
 
 class NetManager(CommandModule):
-    def __init__(self, agent: NetAgent):
+    def __init__(self, agent: NetAgent, diagnose=False):
```

Comment:
How do we turn this on in a live system?

Reply:
We don't, since it will turn on some dangerous commands like stop_net, stop_cell, etc.

Reply:
We need to make it configurable in server/admin.py to turn it on / off when creating the NetManager? Otherwise, we may not be able to use this?

Reply:
Commands are invisible but still available to users.

Reply:
For FLARE, NetManager will always run in non-diagnose mode, which will disable dangerous commands (stop_cell, stop_net), and make other useful commands invisible (but still accessible).

The NetManager will be used in diagnose mode only in the independent cellnet testing tools (QAT stuff).